<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MeshList</class>
 <widget class="QDockWidget" name="MeshList">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>246</width>
    <height>446</height>
   </rect>
  </property>
  <property name="features">
   <set>QDockWidget::DockWidgetFloatable|QDockWidget::DockWidgetMovable</set>
  </property>
  <property name="windowTitle">
   <string>&amp;Meshes</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QListView" name="list"/>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QToolButton" name="showMaterial">
        <property name="toolTip">
         <string>Edit Material…</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../resource/ackui.qrc">
          <normaloff>:/icons/material.png</normaloff>:/icons/material.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="toggleMeshVis">
        <property name="toolTip">
         <string>Hide…</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../resource/ackui.qrc">
          <normaloff>:/icons/eye-off.png</normaloff>:/icons/eye-off.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="openTools">
        <property name="toolTip">
         <string>Apply tool…</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../resource/ackui.qrc">
          <normaloff>:/icons/screwdriver.png</normaloff>:/icons/screwdriver.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QToolButton" name="deleteMesh">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../resource/ackui.qrc">
          <normaloff>:/icons/delete.png</normaloff>:/icons/delete.png</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="../resource/ackui.qrc"/>
 </resources>
 <connections/>
</ui>
